style.scss

```diff
@@ -0,0 +1,73 @@
+* {
+  margin: 0;
+  padding: 0;
+  box-sizing: border-box;
+}
+
+.wrapper {
+  margin: 0 auto;
+}
+
+.keyboard {
+  width: 665px;
+  margin: 0 auto;
+}
+
+.keyboard .row {
+  clear: both;
+  width: 100%;
+  height: 40px;
+  margin-bottom: 5px;
+}
+
+.keyboard .keybutton {
+  float: left;
+  width: 40px;
+  height: 40px;
+  background-color: #000;
+  margin-left: 5px;
+  border-radius:5px;
+  text-align: center;
+  line-height: 40px;
+  font-weight: bold;
+  color: #ffffff;
+  cursor: pointer;
+
+}
+
+.keyboard .keybutton:hover {
+  border: 1px solid #000000;
+  background-color: #ffffff;
+  color: #000000;
+}
+
+.keyboard .shift {
+width: 102px;
```

Comment:
OK in this small app, but in the future you can move many numbers to appropriate variables, and to another file called `_variables.scss`